### DISCUSSION TITLE
Changing base branch with long branch names

### DISCUSSION THREAD

**Discussion by fabianallendorf** [community/community#16317] [Pull Requests]
This happened when I created two branches with long names that shared a prefix. Only the suffix differs. The current UI doesn't allow me to see the complete branch name, so there is no way to know which branch I need to choose. 
 
![image](https://user-images.githubusercontent.com/7349133/167578443-55d8c69e-0597-40b2-9881-42d260b7837e.png)

It would help a lot if it was possible to see the full branch name when mouse hovering over the branch name.

**Comment by lucalves** [comment]
I really think it would also be a good idea to display the branch name when hovering over it.

However, if you go to https://github.com/[YOUR_USER}/[YOUR_REPOSITORY]/branches and click on the branch, you will directly display the code for that specific branch.

🙂
